editor/app/ems/[incident_number]/page.tsx

```diff
@@ -248,13 +296,26 @@ export default function EMSDetailsPage({
             noRowsMessage="No people found"
             header="Possible people matches"
             columns={emsMatchingPeopleColumns}
-            mutation=""
+            mutation={UPDATE_PERSON}
```

Comment:
we didnt have a mutation for this even though some of the columns are editable, you can test on staging and see that when you try to edit one of the editable columns you get an error. so i updated this, unless we dont want to allow editing people records from ems incidents pages? i think its useful though